docs/user_guide/datasets/acs_public_coverage.rst

```diff
@@ -0,0 +1,274 @@
+
+   *  - NATIVITY
+      - Nativity:
+         1. Native
+         2. Foreign born
+         
+   *  - DEAR
+      - Hearing difficulty:
+         1. Yes
+         2. No
+         
+   *  - DEYE
+      - Vision difficulty:
+         1. Yes
+         2. No
+         
+   *  - DREM
+      - Cognitive difficulty:
+         1. Yes
+         2. No
+         
+   *  - PINCP
+      - Total annual income per person as an integer between -19997 and 4209995
+        US dollars. Loss of $19998 or more is coded as -19998. Income of
+        $4209995 or more is coded as 4209995.
+
+   *  - ESR
+      - Employment status recode:
+         1. Civilian employed, at work
+         2. Civilian employed, with a job but not at work
+         3. Unemployed
+         4. Armed forces, at work
+         5. Armed forces, with a job but not at work
+         6. Not in labor force
+         
+   *  - ST
+      - State code: Please see data dictionary at `ACS PUMS documentation
+         <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_
+         for the full list of state codes.
+
+   *  - FER
+      - Gave birth to child within the past 12 months:
+         1. Yes
+         2. No
+
+   *  - RAC1P
+      - Race code
+         1. White alone
+         2. Black or African American alone
+         3. American Indian alone
+         4. Alaska Native alone
+         5. American Indian and Alaska native tribes specified; or American
+            Indian or Alaska Native, not specified and no other races
+         6. Asian alone
+         7. Native Hawaiian and Other Pacific Islander alone
+         8. Some Other Race alone
+         9. Two or More races
+
+
+The target label is given by PUBCOV, which can be used for a binary
+classification task.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 7 30
+   :stub-columns: 1
+
+   *  - Column name
+      - Description
+
+   *  - PUBCOV
+      - Public health coverage, with PUBCOV == 1 if the individual has public
+        health coverage, else 0
+
+
+.. _acs_public_coverage_discussion:
+
+Discussion
+^^^^^^^^^^
+
+A meaningful evaluation of algorithmic (un)fairness considers the context in
+which the machine learning model will be used. In this way, we can avoid
+falling into abstraction traps (see :footcite:`selbst2019fairness`). For
+example, the relationship between demographics and healthcare coverage may vary
+widely by state - a model that works well in one state may not be suitable to
+use in a different state.
+
+To investigate how viable ACSPublicCoverage is as a fairness benchmark dataset,
+we must first ask what real-world tasks might require a prediction of whether
+an individual is covered by public health insurance. Usually, the entity that
+needs this information (e.g., the government checking for eligibility in a
+social program) will simply ask the individual directly. Ding et al. do not
+provide any examples, so it is difficult to evaluate what notion of fairness is
```

Comment:
Definitely agreed that this is difficult for the author of this documentation to do in the abstract, but is the implication also that this would be difficult for users of this dataset?

Reply:
To be honest I am still very confused about the prediction task and I feel like we could maybe expand a bit on why it is important that the task makes sense before we can even talk about fairness.

It reminds me a bit about the "studying up" paper; why are we focussing on low-income individuals. How would something like this be used to help them? Or could this prediction task also be misused by, I don't know, predatory health insurance companies or "fraud" detection by the government (i.e., the government uses the model to check whether people are "honest" about whether they are covered by public health insurance)? I have no idea whether these examples make any sense, because I don't know much about the US context, but reading this paragraph did not clear anything up for me.

@mmadaio @MiroDudik @romanlutz being more familiar with the US context, do you have thoughts?

Reply:
@britneyting @kspieks @imatiach-msft and I were discussing this back in January as well. I share your concerns. In fact, I feel the same about the income prediction and all the other tasks mentioned in the paper. The context in which this task is useful is not obvious to me. That said, I do believe that this is something we can discuss in the user guide. It leads to the bigger question of "Should benchmarks always be realistic?" I have thoughts on that... Can we perhaps push people to more realistic datasets by outlining shortcomings?

Anyway, curious to hear what you and others think, @hildeweerts .

Reply:
we can also open an issue and discuss it further and not block this PR for this.

Reply:
@romanlutz given that you have thought about this extensively, do you have a suggestion to rephrase this paragraph to make it clearer what our concerns are?

I like @adrinjalali's suggestion to open a new issue to discuss the more general problem more thoroughly.

Reply:
I'll have to think about this a bit more to phrase it right. It may take me a while, though. 

I'm not sure whether we should proceed with this anyway (as we already have Adult and the other variation) and open an issue for describing this sort of problem, or block this PR instead. I see the concern (as mentioned earlier), but given that we have others with the same issue it feels like an odd point to block. On the other hand, that opens the door for more and more contributions that don't interrogate this problem (at least until we capture this in some way).

Reply:
I am also not sure... if we block the PR now I don't think it will get merged in the foreseeable future (if ever), but if we do not touch upon this issue at all we also create more "technical" debt.

I think it should be possible to rephrase the current text a bit to make our concerns clearer, without interrogating the problem in detail. E.g., I think some examples can go a long way, e.g., who is using it (e.g., a governmental organization versus industry) and for what purpose (e.g., a model used to distribute benefits versus a model that is used for fraud detection) already makes a huge difference in how we think about fairness.

Reply:
I agree 100% in the general case. 

In this particular one I'm still wondering what that might be. Any government org that needs to know your insurance status can just ask. Same for companies. And if they don't feel like they can ask they probably aren't meant to know and it's a nefarious or at least borderline/grey area activity. Besides, I just can't wrap my head around why one would have ALL THE OTHER features but just not this one? That seems like a significant amount of detail we know about these individuals, how come we know all that and still need to guess the insurance status? Again, if we lawfully knew this much for a good reason they'd probably tell us that status, too. It's entirely possible my reasoning is flawed somewhere, of course.

Reply:
You raise very good points - especially because it is based on such an apparently well-defined concept, the need for a predictive model seems... odd. I'm inclined to agree with your former comment and say we should just not proceed with this at all.

It's a pity of all the work that went into this, but perhaps sunk cost is not the best argument for including the dataset.